redis/tests/support/mod.rs

```diff
@@ -549,6 +550,39 @@ impl TestContext {
         self.client.get_multiplexed_tokio_connection().await
     }
 
+    #[cfg(all(feature = "tokio-comp", feature = "cache-aio"))]
+    pub fn multiplexed_async_connection_tokio_with_cache(
```

Comment:
why is this needed? I see that everywhere you use `multiplexed_async_connection_tokio_with_cache_config` it's with the default cache.